### ISSUE TITLE
Can snippet autocompletion work in markdown?

### ISSUE THREAD

**Issue by vincentmader** [neoclide/coc-snippets#162]
This question might not be limited to snippets only (but coc autocompletion in general), but it seems to me that snippet autocompletion does not work in markdown files.

Do you know whether this a problem with my specific configuration or rather with something else?

**Comment by chemzqm** [comment]
No idea, try minimal vimrc.

**Comment by vincentmader** [comment]
Ok, I just modified my neovim config to contain only the following lines:

```
call plug#begin('~/.config/nvim/plugged')

  Plug 'neoclide/coc.nvim', {'branch': 'release'} 

call plug#end()
```

With this, autocompletion in markdown does not work. Although the completion list does appear when I start typing, pressing Enter only leads to a new line being inserted. Any idea on why this is the case (it works in every other filetype that I tried) or how one could fix this? 

It would be really handy to have this work in markdown for word autocompletion as well as snippets, therefore I am a bit confused as to why this seems not to have been addressed before.
